pkg/cmd/indices/config/import/synonyms.go

```diff
@@ -0,0 +1,70 @@
+package indiceimport
+
+import (
+	"fmt"
+
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/search"
+
+	"github.com/algolia/cli/pkg/cmd/synonyms/shared"
+)
+
+type ImportConfigJson struct {
```

Comment:
I'd have preferred using the same struct for `import` and `export` but `search.Synonym` struct doesn't work when using `json.Unmarshall`: I think it's a bug from how `search.Synonym` define `Unmarshall` but couldn't figure out what causes it (I can't test it locally since everything is related to the read-only package

Imo we should discuss about it